app/src/main/java/eu/kanade/presentation/more/settings/screen/SettingsDataScreen.kt

```diff
@@ -652,6 +657,22 @@ object SettingsDataScreen : SearchableSettings {
     @Composable
     private fun getSelfHostPreferences(syncPreferences: SyncPreferences): List<Preference> {
         val scope = rememberCoroutineScope()
+
+        val qrScanLauncher = rememberLauncherForActivityResult(ScanContract()) {
+            if (it.contents != null && it.contents.isNotEmpty()) {
+                syncPreferences.clientAPIKey().set(it.contents)
+            }
+        }
+
+        val scanOptions = remember {
+            ScanOptions().apply {
+                setDesiredBarcodeFormats(ScanOptions.QR_CODE)
+                setOrientationLocked(false)
+                setPrompt(stringResource(SYMR.strings.scan_qr_code))
+                addExtra(Intents.Scan.SCAN_TYPE, Intents.Scan.MIXED_SCAN)
+            }
+        }
```

Comment:
Forgot about this, make sure to import LocalContext
```suggestion
        val context = LocalContext.current
        val scanOptions = remember {
            ScanOptions().apply {
                setDesiredBarcodeFormats(ScanOptions.QR_CODE)
                setOrientationLocked(false)
                setPrompt(SYMR.strings.scan_qr_code.toString(context))
                addExtra(Intents.Scan.SCAN_TYPE, Intents.Scan.MIXED_SCAN)
            }
        }
```